core/src/main/java/com/linecorp/armeria/server/docs/NamedTypeInfo.java

```diff
@@ -38,11 +35,9 @@ public interface NamedTypeInfo {
     String name();
 
     /**
-     * Returns the description information. If not available, a null value is returned.
+     * Returns the description information. If not available, {@link DescriptionInfo#empty()} is returned.
```

Comment:
Would it make sense to make this method as a default method that returns `DescriptionInfo#empty()`?

Reply:
`DescriptionInfo` is an essential part of `NamedTypeInfo`. I'd like to enforce subtypes to implement it.

Reply:
That's reasonable. 👍 
There probably are no users that implement this interface by themselves, so it shouldn't be a problem.